zulip_bots/zulip_bots/bots/trivia_quiz/trivia_quiz.py

```diff
@@ -198,14 +199,28 @@ def format_quiz_for_markdown(quiz_id, quiz):
     )
     return content
 
-def grade_question(quiz, answer):
+def grade_question(quiz, answer, quiz_id, bot_handler):
     correct = (answer == quiz['correct_letter'])
+    long_answer = quiz['answers'][quiz['correct_letter']]
 
-    if correct:
-        long_answer = quiz['answers'][answer]
+    if correct and len(quiz['answered_options']) <= 2:
         response = '**CORRECT!** {long_answer} :tada:'.format(long_answer=long_answer)
```

Comment:
I think I would simplify this logic.  If you get a correct answer, don't worry about whether we already revealed the correct answer.  It's possible that somebody answered before seeing the correct answers.